packages/graphql/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
```

Comment:
would be good to extend the top level `tspconfig.base.json` and leave any overrides